firewall.go

```diff
@@ -0,0 +1,246 @@
+package cloudflare
+
+import (
+	"encoding/json"
+	"net/url"
+	"strconv"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+// AccessRule represents a firewall access rule.
+type AccessRule struct {
+	ID            string                  `json:"id,omitempty"`
+	Notes         string                  `json:"notes,omitempty"`
+	AllowedModes  []string                `json:"allowed_modes,omitempty"`
+	Mode          string                  `json:"mode,omitempty"`
+	Configuration AccessRuleConfiguration `json:"configuration,omitempty"`
```

Comment:
Please run gofmt on this file.

Reply:
Is there something you're objecting to on this file? When I run gofmt I see no changes.

Reply:
@SamWhited What diff are you seeing here? This looks fine to me; gofmt uses spaces to align struct fields/tags. Travis isn't reporting an error.

Reply:
My apologies, the fields of this struct don't appear to be vertically aligned but maybe GitHub just isn't using a fixed width font. When I open it locally everything looks fine.